libs/program_options/src/convert.cpp

```diff
@@ -79,7 +79,7 @@ namespace hpx { namespace program_options { namespace detail {
 
 namespace hpx { namespace program_options {
 
-    HPX_EXPORT std::wstring from_8_bit(const std::string& s,
+    std::wstring from_8_bit(const std::string& s,
```

Comment:
Not sure if this is correct. Are those symbols not needed outside of our shared libraries? OTOH, this could be a leftover from converting this library from Boost, where these symbols might have been reused by other libraries.

Reply:
It's exported in the header file: https://github.com/STEllAR-GROUP/hpx/blob/62c825ff76ffc3a21588ca2c36730c9c9753aa5a/libs/program_options/include/hpx/program_options/detail/convert.hpp#L41-L42. That export might not be needed of course, bit it's definitely not needed here.